kalohq/kalo-ui-helpers
# Update dependencies

#### How to verify

1. Check out this branch
1. `bundle install`
1. `rails db:migrate`
1. `rake`
1. Observe that all tests pass
